tensilelite/Tensile/Tests/common/gemm/dtv.yaml

```diff
@@ -455,10 +458,11 @@ BenchmarkProblems:
         - LdsPadA: [-1]
         - LdsPadB: [-1]
         - 1LDSBuffer: [1]
+        - TransposeLDS: [0,1]
         #- GlobalSplitU: [1,2]
         #- GlobalSplitUAlgorithm: ["MultipleBuffer", "MultipleBufferSingleKernel"]
         - SourceSwap: [0, 1]
-        - LocalReadVectorWidth: [4]#[4,8]
+        - LocalReadVectorWidth: [2,4,8]
```

Comment:
2 will not be used if TransposeLDS=0 is rejected.

Reply:
Got it. So I think TLDS=0 is rejected in the PR, this should not cause problems.